File: Lab 6/src/main/java/Penthouse.java
public class Penthouse extends Accommodation {
    private boolean basin;

    public Penthouse(boolean basin, int numberOfRooms, double price, double area, double latitude, double height,
                     double longitude) {
        super(numberOfRooms, price, area, latitude, height, longitude);
        this.basin = basin;
    }

    @Override
    public String toString() {
        return "\n\nPenthouse:"
                + "\nbasin = " + basin
                + "\nlocation = " + getLocation()
                + "\nnumber of rooms = " + getNumberOfRooms()
                + "\nprice = " + getPrice()
                + "\narea = " + getArea();
    }

    public boolean isBasin() {
        return basin;
    }

    public void setBasin(boolean basin) {
        this.basin = basin;
    }


}

File: Lab 10 (lab 1 python)/penthouse.py
from accommodation import Accommodation


class Penthouse(Accommodation):
    def __init__(self, number_of_rooms, price, area, latitude, basin):
        super(Penthouse, self).__init__(number_of_rooms, price, area, latitude)
        self.basin = basin

    def __str__(self):
        return super(Penthouse, self).__str__() + " basin = " + str(self.basin)